pkgs/development/compilers/scryer-prolog/cargo.patch

```diff
@@ -1,66 +1,24 @@
 diff --git a/Cargo.lock b/Cargo.lock
-index 4d699f11..1eb894fc 100644
+index 05e54911..3a567165 100644
 --- a/Cargo.lock
 +++ b/Cargo.lock
-@@ -875,15 +875,6 @@ version = "0.1.4"
- source = "registry+https://github.com/rust-lang/crates.io-index"
- checksum = "28988d872ab76095a6e6ac88d99b54fd267702734fd7ffe610ca27f533ddb95a"
- 
--[[package]]
--name = "openssl-src"
--version = "300.0.2+3.0.0"
--source = "registry+https://github.com/rust-lang/crates.io-index"
--checksum = "14a760a11390b1a5daf72074d4f6ff1a6e772534ae191f999f57e9ee8146d1fb"
--dependencies = [
-- "cc",
--]
--
- [[package]]
- name = "openssl-sys"
- version = "0.9.70"
-@@ -893,7 +884,6 @@ dependencies = [
-  "autocfg 1.0.1",
-  "cc",
-  "libc",
-- "openssl-src",
-  "pkg-config",
-  "vcpkg",
- ]
-@@ -1432,6 +1422,7 @@ dependencies = [
-  "divrem",
+@@ -1826,6 +1826,7 @@ dependencies = [
+  "futures",
   "fxhash",
   "git-version",
 + "gmp-mpfr-sys",
   "hostname",
-  "indexmap",
-  "instructions-template",
+  "hyper",
+  "hyper-tls",
 diff --git a/Cargo.toml b/Cargo.toml
-index 8e81a947..d21a8d49 100644
+index 6e42de23..a50e3d47 100644
 --- a/Cargo.toml
 +++ b/Cargo.toml
-@@ -55,7 +55,7 @@ ring = "0.16.13"
- ripemd160 = "0.8.0"
- sha3 = "0.8.2"
- blake2 = "0.8.1"
--openssl = { version = "0.10.29", features = ["vendored"] }
-+openssl = { version = "0.10.29" }
```

Comment:
The patch must be regenerated to remove this line.

Reply:
The patch was regenerated, and this is the diff between the last patch and the current. All of these lines here are being removed from the patch, so the additon and the removal of openssl are both being removed from the patch, and there is no reference to openssl in the Cargo.toml resulting patch file with the diff applied.

Reply:
You are right again.